base/linalg/arnoldi.jl

```diff
@@ -296,6 +296,12 @@ end
 
 ## v = [ left_singular_vector; right_singular_vector ]
 *{T,S}(s::SVDOperator{T,S}, v::Vector{T}) = [s.X * v[s.m+1:end]; s.X' * v[1:s.m]]
```

Comment:
I don't think this is needed anywhere else. It was just used by the `eigs` function so no harm should arise from removing it.